src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -215,12 +242,19 @@ Graph QuantizeGraph(Graph &&src) {
           min_index = num_outputs + 2 * e.index;
           max_index = num_outputs + 2 * e.index + 1;
         } else {
-          CHECK(mirror_node->op() != nullptr &&
-                mirror_node->op()->name == "_contrib_quantize")
+          CHECK((mirror_node->op() != nullptr &&
+                mirror_node->op()->name == "_contrib_quantize") ||
```

Comment:
Same here replace the op name by a function

Reply:
Same comments as above.

Reply:
See reply above